charts/linkerd2/templates/hook-pre-install-linkerd-config.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
     {{.ControllerNamespaceLabel}}: {{.Namespace}}
   annotations:
     {{.CreatedByAnnotation}}: {{default (printf "linkerd/helm %s" .LinkerdVersion) .CliVersion}}
+    "helm.sh/hook": pre-install
```

Comment:
also, I am not sure we need these annotations if we are not using helm. You can check with 
`{{- if empty .CliVersion }}` to make sure that this is not a helm installation.